core/src/test/kotlin/com/karumi/kotlinsnapshot/core/Utils.kt

```diff
@@ -0,0 +1,18 @@
+package com.karumi.kotlinsnapshot.core
+
+import java.io.File
+import java.nio.file.Paths
+
+internal object TestCaseExtractorNotSupported : TestCaseExtractor() {
+    override fun getTestStackElement(): StackTraceElement? = null
+}
+
+fun getRootFile(): File = File("__snapshot__")
```

Comment:
I don't like the idea of polluting the global namespace with these functions, even for tests. What about creating a `SnapshotFileReader` interface with these methods?

Reply:
yep, it can be done with both :). The enclosing scope for an object is the object itself so you call them statically by using the object's name. For interfaces is even cooler, since you declare them into an interface and then just the classes that implement that interface are able to get those extfuns displayed into the autocomplete namespace 👌 

It's somehow similar to how you can enable features by trait mixin in Scala, where the trait's implemented are the enclosing scopes.